### PR TITLE
Completed Input, Navigation and Rating Component

### PR DESCRIPTION
**Pull request by varun1233**
Completed Input, Navigation and Rating Component
image
image
image
image

### CODE REVIEW

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  margin-right: 1.5rem;
+}
+.search-bar {
+  display: flex;
+  width: 25rem;
+  padding: 0.75rem;
+  border-radius: 4px;
+  background-color: var(--secondary-font-color);
+}
+.search-input {
```
Comment (AshutoshM29) [inline components/navigation/navigation.css:22]: You can remove the outline from the input

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
Looks good to me!
Can be merged

**Review by parvez-rashid — APPROVED** [review]
The site looks great. I couldn't find any irregularities, code looks good to merge.

File: components/navigation/navigation.css (hunk 1 above)
Comment (parvez-rashid) [reply components/navigation/navigation.css:22]: Actually removing the outline (i.e. using `outline: none`) is bad for accessibility.

Comment (mehulsatardekar) [reply components/navigation/navigation.css:22]: probably you can make outline:0.

**Review by mehulsatardekar — APPROVED** [review]
Looks good to merge @varun1233. Didn't find any issue, love that you have followed best practices for writing CSS.

File: components/navigation/navigation.css (hunk 1 above)
Comment (varun1233) [reply components/navigation/navigation.css:22]: Done

**Pull request merged**